Source/BugsnagConfiguration.m

```diff
@@ -199,6 +200,20 @@ - (void)setAppVersion:(NSString *)newVersion {
     }
 }
 
+@synthesize apiKey = _apiKey;
+
+- (NSString *)apiKey {
+    return _apiKey;
+}
```

Comment:
could we remove this as it is just the default generated by `@synthesize`?

Reply:
I think it needs to be specified since `setApiKey` is overridden.